<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AQModBus</class>
 <widget class="QMainWindow" name="AQModBus">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>312</width>
    <height>450</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>AQModBus</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>aqmodbus.png</normaloff>aqmodbus.png</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(0, 0, 0);
border-color: rgb(133, 192, 24);
color: rgb(255, 255, 255);</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QPushButton" name="closeBtn">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>350</y>
      <width>101</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-style: solid;
border-color: rgb(133, 192, 24);
border-width: 1px;
font-weight: bold</string>
    </property>
    <property name="text">
     <string>Close</string>
    </property>
    <property name="checkable">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="settingsBtn">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>350</y>
      <width>101</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-style: solid;
border-color: rgb(133, 192, 24);
border-width: 1px;
font-weight: bold</string>
    </property>
    <property name="text">
     <string>Settings</string>
    </property>
    <property name="checkable">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="connectBtn">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>350</y>
      <width>101</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-style: solid;
border-color: rgb(133, 192, 24);
border-width: 1px;
font-weight: bold</string>
    </property>
    <property name="text">
     <string>Connect</string>
    </property>
    <property name="checkable">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>311</width>
      <height>341</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="tabCoils">
     <attribute name="title">
      <string>Coils</string>
     </attribute>
     <widget class="QProgressBar" name="pbCoil">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>0</y>
        <width>20</width>
        <height>20</height>
       </rect>
      </property>
      <property name="maximum">
       <number>1</number>
      </property>
      <property name="value">
       <number>0</number>
      </property>
      <property name="format">
       <string/>
      </property>
     </widget>
     <widget class="QLedMatrix" name="CoilMatrix" native="true">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>0</y>
        <width>291</width>
        <height>221</height>
       </rect>
      </property>
      <property name="backgroundColor" stdset="0">
       <color>
        <red>0</red>
        <green>0</green>
        <blue>0</blue>
       </color>
      </property>
      <property name="darkLedColor" stdset="0">
       <color>
        <red>34</red>
        <green>34</green>
        <blue>34</blue>
       </color>
      </property>
      <property name="rows" stdset="0">
       <number>8</number>
      </property>
      <property name="columns" stdset="0">
       <number>8</number>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tabInput">
     <attribute name="title">
      <string>Input</string>
     </attribute>
     <widget class="QLedMatrix" name="InputMatrix" native="true">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>0</y>
        <width>291</width>
        <height>221</height>
       </rect>
      </property>
      <property name="backgroundColor" stdset="0">
       <color>
        <red>0</red>
        <green>0</green>
        <blue>0</blue>
       </color>
      </property>
      <property name="darkLedColor" stdset="0">
       <color>
        <red>34</red>
        <green>34</green>
        <blue>34</blue>
       </color>
      </property>
      <property name="rows" stdset="0">
       <number>8</number>
      </property>
      <property name="columns" stdset="0">
       <number>8</number>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tabReg">
     <attribute name="title">
      <string>Registers</string>
     </attribute>
     <widget class="QTableWidget" name="RegisterList">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>1</y>
        <width>141</width>
        <height>211</height>
       </rect>
      </property>
      <property name="showGrid">
       <bool>true</bool>
      </property>
      <property name="rowCount">
       <number>1</number>
      </property>
      <property name="columnCount">
       <number>2</number>
      </property>
      <attribute name="verticalHeaderVisible">
       <bool>false</bool>
      </attribute>
      <row/>
      <column>
       <property name="text">
        <string>Addr</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Register Value</string>
       </property>
      </column>
     </widget>
     <widget class="QTableWidget" name="AnalogList">
      <property name="geometry">
       <rect>
        <x>160</x>
        <y>0</y>
        <width>141</width>
        <height>211</height>
       </rect>
      </property>
      <property name="rowCount">
       <number>1</number>
      </property>
      <property name="columnCount">
       <number>2</number>
      </property>
      <attribute name="verticalHeaderVisible">
       <bool>false</bool>
      </attribute>
      <row/>
      <column>
       <property name="text">
        <string>Addr</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Analog Input</string>
       </property>
      </column>
     </widget>
    </widget>
    <widget class="QWidget" name="tabWrite">
     <attribute name="title">
      <string>Write</string>
     </attribute>
     <widget class="QComboBox" name="cbWrite">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>281</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(133, 192, 24);</string>
      </property>
      <item>
       <property name="text">
        <string>Coil</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Register</string>
       </property>
      </item>
     </widget>
     <widget class="QWidget" name="gridLayoutWidget">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>50</y>
        <width>291</width>
        <height>91</height>
       </rect>
      </property>
      <layout class="QGridLayout" name="gridLayout">
       <item row="2" column="1">
        <widget class="QLineEdit" name="WriteValue">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>40</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">border-style: solid;
border-color: rgb(133, 192, 24);
border-width: 1px;</string>
         </property>
        </widget>
       </item>
       <item row="2" column="3">
        <widget class="QPushButton" name="btnWrite">
         <property name="minimumSize">
          <size>
           <width>90</width>
           <height>40</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">border-style: solid;
border-color: rgb(133, 192, 24);
border-width: 1px;</string>
         </property>
         <property name="text">
          <string>Write</string>
         </property>
         <property name="checkable">
          <bool>false</bool>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QLabel" name="lValue">
         <property name="text">
          <string>Value</string>
         </property>
        </widget>
       </item>
       <item row="0" column="0">
        <widget class="QLabel" name="lAddress">
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>30</height>
          </size>
         </property>
         <property name="text">
          <string>Address</string>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QLineEdit" name="WriteAddress">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>40</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">border-style: solid;
border-color: rgb(133, 192, 24);
border-width: 1px;</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>QLedMatrix</class>
   <extends>QWidget</extends>
   <header>qledmatrix.h</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>settingsBtn</tabstop>
  <tabstop>connectBtn</tabstop>
  <tabstop>closeBtn</tabstop>
  <tabstop>WriteAddress</tabstop>
  <tabstop>WriteValue</tabstop>
  <tabstop>btnWrite</tabstop>
  <tabstop>cbWrite</tabstop>
  <tabstop>AnalogList</tabstop>
  <tabstop>RegisterList</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
